db/migrate/20160319110950_update_user_reputation_from_existing_votes.rb

```diff
@@ -0,0 +1,31 @@
+class UpdateUserReputationFromExistingVotes < ActiveRecord::Migration
```

Comment:
we already have users on the platform, who have asked or answered or have upvoted question/answers/comments, however there is supposed to be a reward for this. Hence the reason we require this migration, especially to ensure consistency across all the environments(local, staging and production). @andela-iokonkwo